packages/react/src/components/Storage/FileUploader/FileUploader/FileUploader.tsx

```diff
@@ -197,7 +195,9 @@ export function FileUploader({
     [fileStatuses, setFileStatuses]
   );
 
-  const onFileClick = useCallback(() => {
+  const startUpload = useCallback(() => {
```

Comment:
Should this be `uploadFile`? `startUpload` reads like a boolean imo

Reply:
good call, I like uploadFile better